libs/wingsdk/src/target-tf-aws/service.ts

```diff
@@ -0,0 +1,338 @@
+import { writeFileSync } from "fs";
+import { join, resolve } from "path";
+import { Construct } from "constructs";
+import { App } from "./app";
+import * as cloud from "../cloud";
+import * as core from "../core";
+import { createBundle } from "../shared/bundling";
+import { AwsInflightHost, IAwsInflightHost, NetworkConfig, PolicyStatement } from "../shared-aws";
+import { EcsTaskDefinition } from "../.gen/providers/aws/ecs-task-definition";
+import { EcsService } from "../.gen/providers/aws/ecs-service";
+import { RegistryImage } from "../.gen/providers/docker/registry-image";
+import { IamRole } from "../.gen/providers/aws/iam-role";
+import { SecurityGroup } from "../.gen/providers/aws/security-group";
+import { Image } from "../.gen/providers/docker/image";
+import { CloudwatchLogGroup } from "../.gen/providers/aws/cloudwatch-log-group";
+import { IInflightHost } from "../std";
+import { Lazy } from "cdktf";
+import { LiftMap } from "../core";
+
+
+export class Service extends cloud.Service implements IAwsInflightHost {
+  private workdir: string;
+  private wrapperEntrypoint: string;
+  private policyStatments?: any[];
+  private dockerFileName: string;
+  private service: EcsService;
+
+  constructor(
+    scope: Construct,
+    id: string,
+    handler: cloud.IServiceHandler,
+    props: cloud.ServiceProps = {}
+  ) {
+    super(scope, id, handler, props);
+    // We probably need to create the ECR per app (App.of(this).ecr)
```

Comment:
Seems like this is what you are doing no?

Reply:
Yea forgor to remove comment after I did it 😁